src/services/car/__tests__/messageLead.test.ts

```diff
@@ -364,6 +368,78 @@ describe("Car API", () => {
     })
   })
 
+  describe("#fetchDealerWhatsappLeads", () => {
+    const { content, pagination } = PaginatedLeads([
+      SearchWhatsappLeadFactory(),
+    ])
+
+    beforeEach(() => {
+      fetchMock.mockClear()
+      fetchMock.mockResponse(
+        JSON.stringify({
+          content: content,
+          ...pagination,
+        })
+      )
+    })
+
+    it("calls correct endpoint", async () => {
+      await fetchDealerWhatsappLeads({
+        dealerId: 1234,
+        query: {
+          page: 2,
+          size: 7,
+          sort: defaultLeadSort,
+        },
+        options: {
+          accessToken: "DummyTokenString",
+        },
+      })
+
+      expect(fetch).toHaveBeenCalledWith(
+        expect.stringMatching(
+          /\/dealers\/1234\/whats-app-tracking-entries\?page=1&size=7&sort=createdDate%2Cdesc$/
+        ),
+        expect.any(Object)
+      )
+    })
+
+    it("should trow error if accessToken is not passed", async () => {
+      try {
+        await fetchDealerMessageLeads({
+          dealerId: 1234,
+          query: {
+            page: 0,
+            size: 7,
+            sort: {},
+            searchQuery: "",
+          },
+        })
+        expect(fetch).not.toHaveBeenCalled()
+      } catch (err) {
+        expect(err).toBeDefined()
+        expect(err.message).toBeDefined()
+      }
```

Comment:
Oh, now I see why you need to disable the linter. Jest supports testing asynchronous code via helpers. In this case [`.rejects` helper](https://jestjs.io/docs/tutorial-async#rejects) should do the trick.